wlct/tournaments.py

```diff
@@ -3838,36 +3850,33 @@ def update_game_log(self):
                 if team_1:
                     team_2 = TournamentTeam.objects.filter(id=int(team2))
                     if team_2:
-                        log += '<td data-search="{} {} {} {}">{}</td>'.format(team_1[0].clan_league_clan.clan.name, team_2[0].clan_league_clan.clan.name, get_team_data_no_clan(team_1[0]), get_team_data_no_clan(team_2[0]), get_matchup_data(team_1[0], team_2[0]))
-                log += '<td><a href="{}" target="_blank">Game Link</a></td>'.format(game.game_link)
+                        game_log += '<td data-search="{} {} {} {}">{}</td>'.format(team_1[0].clan_league_clan.clan.name, team_2[0].clan_league_clan.clan.name, get_team_data_no_clan(team_1[0]), get_team_data_no_clan(team_2[0]), get_matchup_data(team_1[0], team_2[0]))
+                game_log += '<td><a href="{}" target="_blank">Game Link</a></td>'.format(game.game_link)
 
                 if game.is_finished:
                     finished_text = '<span class="text-success"><b>Finished</b></span>'
                 else:
                     finished_text = '<span class="text-info">{}</span>'.format(game.current_state)
-                log += '<td>{}</td>'.format(finished_text)
+                game_log += '<td>{}</td>'.format(finished_text)
 
                 if game.is_finished:
                     winning_team = '{}'.format(get_team_data(game.winning_team))
                 else:
                     winning_team = ''
-                log += '<td>{}</td>'.format(winning_team)
+                game_log += '<td>{}</td>'.format(winning_team)
                 time_to_boot_calculate = 0
-                if game.game_boot_time != "" and game.game_boot_time is not None:
-                    if game.is_finished:
-                        time_to_boot = game.game_finished_time
-                    elif game.game_boot_time.replace(tzinfo=None) < datetime.datetime.now().replace(tzinfo=None) and game.current_state == "WaitingForPlayers":
-                        time_to_boot = game.game_boot_time.strftime("%b %d, %Y %H:%M:%S %p")
-                        time_to_boot_calculate = 1
-                    else:
-                        time_to_boot = "N/A"
-                else:
-                    time_to_boot = "N/A"
-                log += '<td class="time_to_boot" data-calculate="{}">{}</td>'.format(time_to_boot_calculate, time_to_boot)
-                log += '</tr>'
+                
+                start_time = game.game_start_time.strftime("%b %d, %Y %H:%M:%S %p")
 
-        log += '</tbody></table>'
-        self.game_log = log
+                # game.game_finished_time check is redundant but is done for backwards compability with bad existing data
+                end_time = game.game_finished_time.strftime("%b %d, %Y %H:%M:%S %p") if game.is_finished and game.game_finished_time else 'N/A'
+
+                game_log += '<td>{}</td>'.format(start_time)
```

Comment:
Am I crazy or did you invent another column here?

Reply:
Yea I added two columns to the game log (Start Time and End Time) and removed the Time to boot column.

Reply:
You will also need to add in another <th> tag at the top I thought. Does this render properly?

Reply:
I did add another column in the top.